internal/protocoltest/restxml/serializers.go

```diff
@@ -3,156 +3,204 @@
 package restxml
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"github.com/aws/aws-sdk-go-v2/internal/protocoltest/restxml/types"
 	smithy "github.com/awslabs/smithy-go"
 	"github.com/awslabs/smithy-go/httpbinding"
 	"github.com/awslabs/smithy-go/middleware"
 	smithytime "github.com/awslabs/smithy-go/time"
 	smithyhttp "github.com/awslabs/smithy-go/transport/http"
+	smithyxml "github.com/awslabs/smithy-go/xml"
+	"time"
 )
 
-type awsRestxml_serializeOpRecursiveShapes struct {
+type awsRestxml_serializeOpAllQueryStringTypes struct {
 }
 
-func (*awsRestxml_serializeOpRecursiveShapes) ID() string {
+func (*awsRestxml_serializeOpAllQueryStringTypes) ID() string {
 	return "OperationSerializer"
 }
 
-func (m *awsRestxml_serializeOpRecursiveShapes) HandleSerialize(ctx context.Context, in middleware.SerializeInput, next middleware.SerializeHandler) (
+func (m *awsRestxml_serializeOpAllQueryStringTypes) HandleSerialize(ctx context.Context, in middleware.SerializeInput, next middleware.SerializeHandler) (
```

Comment:
Is this file being generated in stable order? I wouldn't expect the order of these parameters to change.

Reply:
I think they change because the last time this file was committed (long time ago), the order was not stable.